.prettierrc

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Still empty but here we can add formatting rules, that we agree on. single/double quotes, line length, etc...

Reply:
We should add the formatting rules from Canvas UI / Contract explorer if possible, to unify the code among projects.

Reply:
https://github.com/paritytech/substrate-contracts-explorer/blob/master/.prettierrc